qemu/tests/win_virtio_driver_install_by_installer.py

```diff
@@ -23,36 +20,41 @@ def run(test, params, env):
     7) Verify the qemu-ga version match expected version.
     8) Run driver signature check command in guest.
        Verify target driver.
+    9) Run driver function test.
```

Comment:
And @menli820  could you update the steps which is still the old logic, also the repair/update functions, thanks.